File: InPython/pattern4.py
# Takes t as input on how many patterns and takes t times output for printing
def pattern4(n):
    for i in range(1,n+1):
        for j in range(1,i+1):
            print(i," ",end = '')  #Since in python,next print will print on the next line so by this end we are saying not go to the nest line
        print("\n")


t = int(input())
for i in range(t):
    n = int(input())
    pattern4(n)

File: Arrays/Patterns/pattern4.java
// pattern

// 5

// 1
// 2 2
// 3 3 3
// 4 4 4 4
// 5 5 5 5 5

//Takes t as input on how many patterns and takes t times output for printing
import java.util.Scanner;

class part {
    public void print(int n) {
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= i; j++) {
                System.out.print(i + " ");
            }
            System.out.println();
        }
    }
}

class pattern4 {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        part obj = new part();
        int t = sc.nextInt();
        int n;
        for (int i = 0; i < t; i++) {
            n = sc.nextInt();
            obj.print(n);
        }
        sc.close();
    }
}

// import java.util.Scanner;

// class pattern4 {
// public static void main(String[] args) {
// Scanner sc = new Scanner(System.in);
// int n = sc.nextInt();
// for (int i = 1; i <= n; i++) {
// for (int j = 1; j <= i; j++) {
// System.out.print(i + " ");
// }
// System.out.println();
// }
// }
// }
